File: 001/Euler1.js
(function euler1() {
  var sum = 0;
  for (var x = 0; x<1000; x++){
    if (x % 3 === 0 || x % 5 === 0) {
      sum += x;
    }
  }
  console.log(sum);
  return sum;
})();

File: 001/Euler1.py
__author__ = 'eddie'


def euler1(num_max):
    temp_list =[]
    for x in range(100):
        if x%5==0:
            temp_list.append(x)
        elif x%3==0:
            temp_list.append(x)
    y = sum(temp_list)
    return y

print euler1(100)


print sum([x for x in range(100) if x%3==0 or x%5==0])
